addons/website_payment/static/tests/tours/donation.js

```diff
@@ -0,0 +1,122 @@
+    const wTourUtils = require("website.tour_utils");
+
+    tour.register("donation_snippet_edition", {
+        test: true,
+        url: "/?enable_editor=1"
+    }, [
+        wTourUtils.dragNDrop({
+            id: "s_donation",
+            name: "Donation"
+        }),
+        ...wTourUtils.clickOnSave(),
+        // -- Testing the minimum amount --
+        {
+            content: "Enter a negative custom amount, testing the minimum amount",
+            trigger: "#s_donation_amount_input",
+            run: "text -5"
+        },
+        {
+            content: "Donate with custom amount set",
+            trigger: ".s_donation_donate_btn",
+            run: "click"
+        },
+        {
+            content: "Check if alert-danger element exists",
+            trigger: "p.alert-danger"
+        },
+        // -- End of testing the minimum amount --
+        {
+            content: "Enter a custom amount",
+            trigger: "#s_donation_amount_input",
+            run: "text 55"
+        },
+        {
+            content: "Donate with custom amount set",
+            trigger: ".s_donation_donate_btn",
+            run: "click"
+        },
+        {
+            content: "Check if custom amount radio input is selected and value is 55",
+            trigger: "input#other_amount",
+            run: function(action) {
+                const input = action.tip_widget.$anchor[0];
+                const inputValue = input.nextElementSibling.children[0].value;
+                if (!input.checked || inputValue !== "55.0") {
+                    console.error("The custom amount radio input is not selected or the value is not 55.");
+                }
+            }
+        },
+        {
+            content: "Select the amount of 25",
+            trigger: "input#amount_1",
+            run: "click"
+        },
+        {
+            content: "Verify that only one input is checked",
+            trigger: "input#amount_1",
+            run: function(action) {
+                const amount1 = action.tip_widget.$anchor[0];
+                const amountDiv = amount1.parentElement.parentElement;
+                const radiosInputs = amountDiv.querySelectorAll("input[type=radio]");
+                const selectedInputs = Array.from(radiosInputs).filter(radio => radio.checked);
+
+                if (selectedInputs.length !== 1) {
+                    console.error("More than one input is checked.");
+                }
+            }
+        },
+        {
+            content: "Change custom amount to 67",
+            trigger: "input#other_amount",
+            run: function(action) {
+                const input = action.tip_widget.$anchor[0];
+                const customInput = input.nextElementSibling.children[0];
+                customInput.value = "67";
+                customInput.dispatchEvent(new Event("input", { bubbles: true }));
+                customInput.dispatchEvent(new Event("change", { bubbles: true }));
+            }
```

Comment:
Try doing it with a `run: "text 67",`, it is supposed to simulate the whole sequence of events.